packages/firebase_common/firebase_common/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: firebase_common
+description: "Firebase Common"
+version: 0.0.1
+publish_to: none
+environment:
+  sdk: '>=3.2.0 <4.0.0'
+  flutter: ">=3.3.0"
+dependencies:
+  flutter:
+    sdk: flutter
```

Comment:
Let's don't make a plugin package for this. I think we can add such support utility functions into firebase_core.